cli_reference/opm/cli-opm-ref.adoc

```diff
@@ -14,6 +14,11 @@ The `opm` command-line interface (CLI) is a tool for creating and maintaining Op
 $ opm <command> [<subcommand>] [<argument>] [<flags>]
 ----
 
+[WARNING]
+====
+The `opm` CLI is not forward compatible. The version of the `opm` CLI used to generate catalog content must be earlier than or equal to the version used to serve the content on a cluster.
```

Comment:
Note to peer reviewer: 
As the term "forward compatible" is used throughout the OCP and RHEL docs, I am going to let this stand for now. IMO, I since this admonition is talking about compatibility between different versions of `opm` (one to generate catalog contents and one to serve the content) as opposed to a software version being compatible with an operating system, this recommendation is more confusing if we go with the ISG's recommendation. 

Let me know your thoughts.